### PR TITLE
Feature/40 frontend grundstruktur

### PR DESCRIPTION
**Pull request by AlexSchmitz222**
Added Frontend Base Structure

luks bjutiful

### CODE REVIEW

File: sl-client/src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.header-menu {
+
+}
```
Comment (JayReturns) [inline sl-client/src/app/app.component.css:3]: Seems unneccesary

File: sl-client/src/app/components/header-menu/header-menu.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<mat-toolbar color="primary">
+  <span>SteakLasagne</span>
+
+  <span class="example-spacer"></span>
```
Comment (JayReturns) [inline sl-client/src/app/components/header-menu/header-menu.component.html:4]: pls no example

File: sl-client/src/app/components/header-menu/header-menu.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-header-menu',
+  templateUrl: './header-menu.component.html',
+  styleUrls: ['./header-menu.component.css']
+})
+export class HeaderMenuComponent implements OnInit {
```
Comment (JayReturns) [inline sl-client/src/app/components/header-menu/header-menu.component.ts:8]: Remove oninit if not necessary

File: sl-client/src/app/components/header-menu/header-menu.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (JayReturns) [inline sl-client/src/app/components/header-menu/header-menu.component.ts:12]: Same here

File: sl-client/src/app/components/transaction-card/transaction-card.component.ts [hunk 5]
```diff
@@ -3,10 +3,10 @@ import {Transaction} from "../../models/transaction.model";
 
 @Component({
   selector: 'expense-card-list',
```
Comment (JayReturns) [inline sl-client/src/app/components/transaction-card/transaction-card.component.ts:5]: Maybe also change?

File: sl-client/src/app/components/transaction-list/transaction-list.component.ts [hunk 6]
```diff
@@ -4,10 +4,10 @@ import {Transaction} from "../../models/transaction.model";
 
 @Component({
   selector: 'expense-list',
```
Comment (JayReturns) [inline sl-client/src/app/components/transaction-list/transaction-list.component.ts:6]: Maybe also change?

**Review by JayReturns — CHANGES_REQUESTED** [review]

File: sl-client/src/app/app.component.css (hunk 1 above)
Comment (AlexSchmitz222) [reply sl-client/src/app/app.component.css:3]: will have a use case later on

File: sl-client/src/app/components/header-menu/header-menu.component.html (hunk 2 above)
Comment (AlexSchmitz222) [reply sl-client/src/app/components/header-menu/header-menu.component.html:4]: oops.

**Review by JayReturns — APPROVED** [review]
🙂

**Pull request merged**